content/en/blog/_posts/2018-05-25-Airflow-Kubernetes-Operator.md

```diff
@@ -114,26 +112,90 @@ Use Travis or Jenkins to run unit and integration tests, bribe your favorite tea
 
 ### 2: CI/CD via Jenkins -> Docker Image
 
-There are a multitude on articles on [generating Docker files within a Jenkins build](https://getintodevops.com/blog/building-your-first-docker-image-with-jenkins-2-guide-for-developers). It's a good rule of thumb that you should never use a user-generated Docker image in a production build. By reserving release tags to the Jenkins user, you can ensure that malicious or untested code will never be run by your production Airflow instances.
+There are a multitude on articles on [generating Docker files within a Jenkins build](https://getintodevops.com/blog/building-your-first-Docker-image-with-jenkins-2-guide-for-developers). It's a good rule of thumb that you should never use a user-generated Docker image in a production build. By reserving release tags to the Jenkins user, you can ensure that malicious or untested code will never be run by your production Airflow instances.
```

Comment:
Shall we skip the CI/CD specific part in such detail? It doesn't seem full fledged enough (underspecified in terms of how one can react to the webhook etc) and the example is too simplistic . Maybe we just add a small section about how one *could* use it for CI/CD?